packages/site/tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "extends": "../tsconfig/react-library.json",
-  "include": [".", "../jupyter/src/execute", "../jupyter/src/controls"],
+  "include": ["."],
```

Comment:
hmm wonder if this was a vscode auto thing, as i didnt; add these explicitly

Reply:
Yeah, when you drag from one folder to another it happens!